### DISCUSSION TITLE
When will 1.0.0-preview12 be released

### DISCUSSION THREAD

**Discussion by jhzhu89** [dotnet/yarp#1020] [General] [answered]
Hi, Thanks for the cool project! 

I am currently using YARP in my service, and I am facing an issue which is already resolved in #951. I just want to ask when will you cut a new release?

**Comment by Tratcher** [comment]
Have you tried using the daily builds to get unblocked?
https://github.com/microsoft/reverse-proxy/blob/main/docs/DailyBuilds.md

We're in the middle of some API cleanup and would like to finish that before the next preview.

**Reply by samsp-msft** [reply]
We are trying to minimize the number of breaking change releases.

**Comment by karelz** [comment, marked as answer]
It was released on 6/30 -- https://github.com/microsoft/reverse-proxy/releases/tag/v1.0.0-preview12
